pubtools/_pulp/tasks/set_maintenance/set_maintenance_on.py

```diff
@@ -34,7 +34,7 @@ def adjust_maintenance_report(self, report):
         if self.args.repo_url_regex:
             # search distributors with relative_url, get the repo id from distributors
             crit = Criteria.with_field(
-                "relative_url", Matcher.regex(self.args.repo_url_regex)
+                "relative_url", Matcher.regex(self.args.repo_url_regex.pattern)
```

Comment:
We can see here that no autotest update was needed to fix this, which is a concern as it means the same bug could happen again and nothing is protecting against it.

I think we should fix that in pulplib - https://github.com/release-engineering/pubtools-pulplib/pull/68/files.  That would avoid the need for an autotest here.

Reply:
Thanks for this :)